SQF/dayz_code/system/zombie_agent.fsm

```diff
@@ -472,7 +472,8 @@ class FSM
        "" \n
        "//Move to target" \n
        "_agent moveTo _targetPos;" \n
-       "_agent forceSpeed 10;" \n
+       "_speed = if (!isNil ""DZE_slowZombies"" && {DZE_slowZombies}) then { 2 } else { 10 };" \n
```

Comment:
Correct, braces are only needed to switch on lazy evaluation. However, in this case lazy evaluation will be slower because (!isNil "DZE_slowZombies") is going to return true the majority of the time (since it is defined in variables.sqf). Lazy evaluation should not be used here.

Lazy evaluation should always be faster, but sometimes in SQF it is not. It is faster in <b>OR</b> cases if one of the first conditions is likely to return true often. And it is faster in <b>AND</b> cases if one of the first conditions is likely to return false often. In other cases it is slower. This is because SQF is dumb and {code} evaluates slightly slower than (condition). See: https://community.bistudio.com/wiki/Code_Optimisation#Conditions